src/components/UserUpdateGroups.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <v-card outlined class="subgroup" :disabled="!adminUser">
+    <h2>User Groups</h2>
+    <v-row no-gutters>
+      <v-col class="col-4">
+        <v-checkbox
+            hide-details="auto"
+            v-for="(group, index) in allGroups"
+            :id="'group-' + index"
+            v-model="selectedGroups"
+            :value="group"
+            :label="group.name"
+            :key="group.id"
+        ></v-checkbox>
+      </v-col>
+    </v-row>
+    <div class="my-6">
+      <v-btn id="save-user-groups" class="secondary"
+             medium v-on:click="updateUserGroups()"
+             v-if="adminUser">
+        Save User Groups
+      </v-btn>
+    </div>
+  </v-card>
+</template>
+
+<script>
+import UsersRepository from "@/api/UsersRepository";
+import GroupsRepository from "@/api/GroupsRepository";
+
+export default {
+  name: "UserUpdateGroups",
+  props: ['userId'],
+  data() {
+    return {
+      adminUser: false,
+      joinedGroups: [],
```

Comment:
Just a suggestion. Add doc, "Joined groups are the user's current groups". Or maybe just rename it `currentGroups`?

Reply:
Agree, I'll update that. Joined groups is the terminology in the Keycloak admin console, but currentGroups is probably more clear